### PR TITLE
refresh task onclick

### PR DESCRIPTION
**Pull request by katzio**

### CODE REVIEW

File: app/src/main/res/layout/earn_tab.xml [hunk 1]
```diff
@@ -24,16 +24,16 @@
 
         <TextView
```
Comment (shayb) [inline app/src/main/res/layout/earn_tab.xml]: need to add loader on the button

File: app/src/main/res/layout/earn_tab.xml [hunk 2]
```diff
@@ -24,16 +24,16 @@
...
             android:id="@+id/balance_text"
```
Comment (shayb) [inline app/src/main/res/layout/earn_tab.xml]: need to disable clicks on the button so no double clicks

File: app/src/main/res/layout/earn_tab.xml (hunk 1 above)
Comment (katzio) [reply app/src/main/res/layout/earn_tab.xml]: done

File: app/src/main/res/layout/earn_tab.xml (hunk 2 above)
Comment (katzio) [reply app/src/main/res/layout/earn_tab.xml]: done

**Pull request merged**